File: backend/art-galery/src/main/java/com/unmsm/movil/tecnisis/art_galery/infrastructure/adapters/input/rest/model/response/RequestResponse.java
package com.unmsm.movil.tecnisis.art_galery.infrastructure.adapters.input.rest.model.response;

import lombok.*;

import java.time.LocalDate;

@Builder @Getter @Setter
@NoArgsConstructor @AllArgsConstructor
public class RequestResponse {
    private Long id;
    private LocalDate date;
    private String status;
    private ArtWorkResponse artWork;
}

File: tecnisis-app/app/src/main/java/com/example/tecnisis/data/request/RequestResponse.kt
package com.example.tecnisis.data.request

import com.example.tecnisis.data.artwork.ArtworkResponse
import com.google.gson.annotations.SerializedName

data class RequestResponse(
    val id: Long,
    val date: String,
    val status: String,
    @SerializedName("art_work")
    val artWork: ArtworkResponse
)
